pkg/autodiscover/autodiscover.go

```diff
@@ -122,16 +113,30 @@ func DoAutoDiscover(config *configuration.TestConfiguration) DiscoveredTestData
 		logrus.Fatalf("Failed to retrieve storageClasses - err: %v", err)
 	}
 
+	// if using only old labels, initialize maps
+	if config.PodsUnderTestLabels == nil {
+		config.PodsUnderTestLabels = make(map[string]string)
+	}
+	if config.OperatorsUnderTestLabels == nil {
+		config.OperatorsUnderTestLabels = make(map[string]string)
+	}
+	// consolidate pods labels
+	for _, aLabel := range config.TargetPodLabels {
```

Comment:
It would be nice showing a warning trace in case the old config for labels is used:
```
if len(config.TargetPodLabels) > 0 {
  logrus.Warnf("DEPRECATED: targetPodLabels config field is about to be obsolete. Please use the new \"podsUnderTestLabels\" field instead. 
}
```
Something similar would be nice for the operators label. What do you think?

Reply:
fixed